src/app/content/studyGuides/hooks/openStudyGuides.ts

```diff
@@ -8,8 +9,13 @@ export const hookBody: ActionHookBody<typeof openStudyGuides> = (services) => as
   const filtersHaveBeenSet = select.filtersHaveBeenSet(state);
   const currentFilters = select.summaryLocationFilters(state);
   const defaultFilter = select.defaultLocationFilter(state);
+  const notLoggedIn = loggedOut(state);
+  const locationFilters = select.studyGuidesLocationFiltersWithContent(state);
+  const firstChapter = Array.from(locationFilters)[0];
```

Comment:
`getHighlightLocationFilterForPage` would work if you passed a chapter into it, you could rename that, pull the first chapter out of some archive util and get the correct chapter one filter that way, rather than adding all the sorting logic to the filter utils, i think that might be cleaner